webapp/src/app/admin/admin.module.ts

```diff
@@ -0,0 +1,36 @@
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { FormsModule } from '@angular/forms';
+
+import { AdminComponent } from './admin.component';
+import { DashboardComponent } from './dashboard/dashboard.component';
+import { AdminRoutingModule } from './admin.routing';
+import { UiModule } from '../ui/ui.module';
+import { SDKBrowserModule } from '../shared/sdk/index';
+
+import { UserModule } from './users/user.module';
+import { RoleModule } from './roles/role.module';
+
+@NgModule({
+  imports: [
+    CommonModule,
+    FormsModule,
+    UiModule,
+    UserModule,
+    RoleModule,
+    SDKBrowserModule,
```

Comment:
Why importing the SDKBrowserModule in here if it is already imported in app.module?